bin/bundle

```diff
@@ -1,3 +1,114 @@
 #!/usr/bin/env ruby
```

Comment:
What is this change? It doesn't appear to have come from running `rails app:update`

Reply:
If I remember correctly, it was suggested while running `rails g` when I tested the generator functionality with rubocop.

The output is the result of running `bundle binstubs bundler --force`.

Reply:
(You can check my comments in the corresponding PR in government-frontend as well.)

Reply:
I did a quick check now, and it seems it doesn't produce the warning for 7.2.1, so I think this part can be reverted.

Reply:
I've reverted the changes to bin/bundle here and in government-frontend as discussed.